.github/workflows/ci.yml

```diff
@@ -97,6 +97,22 @@ jobs:
             }}
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: Upload awx jUnit test reports
+        if: >-
+          matrix.test.name == 'api-test'
+          && steps.make-run.outputs.test-result-files != ''
+          && github.event_name == 'push'
+          && github.repository == 'ansible/awx'
+          && github.ref_name == 'devel'
+        continue-on-error: true
+        run: >-
+          curl -v --user "${{ vars.PDE_ORG_RESULTS_AGGREGATOR_UPLOAD_USER }}:${{ secrets.PDE_ORG_RESULTS_UPLOAD_PASSWORD }}"
+          --form "xunit_xml=${{ steps.make-run.outputs.test-result-files }}"
```

Comment:
Note that `${{ steps.make-run.outputs.test-result-files }}` may contain several files, so you probably need multiple `curl` calls unless the endpoint accepts multiple files in a single playload.

Reply:
Also, add a leading `@`